src/commands/lambda/README.md

```diff
@@ -17,13 +17,13 @@ export AWS_SECRET_ACCESS_KEY="<ACCESS KEY>"
 datadog-ci lambda <command>
 ```
 
-### API
+Download the [Datadog CI][2].
 
-#### Configuration
+### Configuration
 
-Configuration is done via a json file, by default the tool load `datadog-ci.json` which can be overriden through the `--config` argument.
+Configuration is done using a JSON file. By default AWS Lambda loads `datadog-ci.json` which can be overriden through the `--config` argument.
 
-The configuration file structure is the following, all fields are optional:
+Use this configuration file structure:
 
```

Comment:
should we add a table of all of the supported options like we did for the other repos?

Reply:
It's probably unnecessary, since the table from the Commands section probably does a good job of SEO already.